### DISCUSSION TITLE
Turn on buzzer in FAILSAFE mode?

### DISCUSSION THREAD

**Discussion by juju988** [iNavFlight/inav#7239] [Q&A] [answered]
Hi all - is it possible to turn on the buzzer (continuous beeping) in FAILSAFE mode?

I was looking to see if there's a way to connect BEEPER flight mode to a global variable in the Programming tab but can't see how to do that. Or, is there a way to activate the BEEPER flight mode if the FAILSAFE flight mode activates?

The scenario would be if the model loses radio connection (for example lands in a ditch in a field or flies out of range) then FAILSAFE will activate and the buzzer will sound continuously until it's found or runs out of juice.

**Comment by stronnag** [comment, marked as answer]
Happens by default here .
